packages/react-keytips/src/docs/Spec.md

```diff
@@ -0,0 +1,190 @@
+    },
+  });
+
+  return <Checkbox label="checkbox" ref={checkboxRef} />;
+};
+```
+
+Some Fabric components have API inherited from it's previous versions, which was based on [keytipProps](https://developer.microsoft.com/en-us/fluentui#/controls/web/keytips#IKeytipProps) with `KeytipData` being used inside the component. This approach added extra bundle size to components and was replaced in `Link`, `Toggle`, `Checkbox`, `ComboBox`, `Dropdown`, and `SpinButton` components in favour of `useKeytipRef` hook API.
+
+```tsx
+export const KeytipsButtonExample = () => {
+  return (
+    <CompoundButton
+      text="CompoundButton"
+      onClick={handleOnClick}
+      keytipProps={{
+        id: 'CompoundButton',
+        content: '1B',
+        optionalProps: {
+          onExecute: (el: HTMLELement) => el.click();
+        },
+      }}
+    />
+  );
+}
+```
+
+### Northstar (v0)
+
+There is not prior implementation for this component in v0.
+
+## Sample Code
+
+```tsx
+import { Keytips, useKeytipRef, ExecuteKeytipEventHandler } from '@fluentui-contrib/react-keytips';
+
+const App = () => {
+  const onExecute: ExecuteKeytipEventHandler = (ev, ({ targetElement })) => {
+    targetElement.click();
+  };
+
+  const checkboxRef = useKeytipRef({
+    keySequences: ['a'],
+    content: ['A']
+    onExecute,
+  });
+
+  return (
+    <>
+      {/* Should be added once at the top level of your app */}
+      <Keytips />
+      <Checkbox label="checkbox" ref={checkboxRef} />
+    </>
+};
+```
+
+## Variants
+
+### Positioning Variants
+
+The `Keytips` component using [positioning API](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--docs) and can be controlled by `positioning` prop in `useKeytipRef` hook.
+The keytip is positioned below and centered to the target element by default.
+
+![Keytip positioning example](assets/positioning.png)
+
+### Style variants
+
+`Keytips` have two style variants (appearance): `normal` and `inverted` (default).
+
+![Keytip style variants example](assets/style-variants.png)
+
+## API
+
+### Keytips
+
+| Prop Name            | Type                                   | Default         | Description                                                     |
+| -------------------- | -------------------------------------- | --------------- | --------------------------------------------------------------- |
+| `mountNode`          |                                        |                 | Where the Portal children are mounted on DOM.                   |
+| `startSequence`      | `string`                               | `'alt+control'` | Key sequence that will start keytips mode.                      |
+| `returnSequence`     | `string`                               | `'escape'`      | Key sequences that execute the return functionality in keytips. |
+| `exitSequence`       | `string`                               |                 | Key sequences that will exit keytips mode.                      |
```

Comment:
Here's also `escape` default?

Reply:
`escape` is already set for `returnSequence`, so it will exit from keytip mode only on the root level of the Keytips. So for the `exitSequence` we won't probably need a default. It will behave like this -> if keytip mode is on, then pressing `startSequence` again will exit from keytip mode and `escape` behave as I mentioned above. If there will be a need for user for specific key sequence for exit, it's possible to add it via this prop :)